apacheconfig/loader.py

```diff
@@ -137,6 +137,8 @@ def interpolate(value):
     def g_statement(self, ast):
         """Performs postprocessing on a statement. Returns an {option: value} dict.
         """
+        if len(ast) == 1:
+            return {ast[0]: None}
         option, value = ast[:2]
```

Comment:
I assume that other values (2+) of `ast` either never exist or are not important so we drop them here. Also, `ast` can't be empty.